src/app/loadout-builder/generated-sets/GeneratedSet.tsx

```diff
@@ -72,14 +73,16 @@ function GeneratedSet({
     set.armor.every((items) => loadout.items.map((item) => item.id).includes(items[0].id))
   );
 
+  const items = set.armor.map((items) => items[0]);
```

Comment:
Might be worth seeing if this component gets re-rendered a lot, and if so, wrapping this and the `getPower` in a useMemo. If it only really re-renders when those would be different anyway, don't bother.

Reply:
K will do